index.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Concert-website</title>
+    <link rel="stylesheet" href="./main.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/js/bootstrap.bundle.min.js"></script>
+
+</head>
+
+<body>
+    <section class="top-nav">
+        <div></div>
+        <div class="top-nav-content">
+            <img src="./items/facebook-icon.png" alt="Facebook icon">
+            <img src="./items/twitter-icon.png" alt="twiter icon">
+            <a href="">English</a>
+            <a href="">My Page</a>
+            <a href="">Logout</a>
+        </div>
+    </section>
+    <header>
+        <a href="index.html" class="logo">
+            <img src="./items/Logo.png" alt="this is the image of the logo">
+        </a>
+        <div class="hamburger">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+        </div>
+        <div class="nav-lists">
+            <a href="about.html" class="nav-link">About</a>
+            <a href="" class="nav-link">Schedule</a>
+            <a href="" class="nav-link">Contact-Us</a>
+            <button>Get Ticket</button>
+
+        </div>
+    </header>
+    <!-- first contents -->
+    <section class="intro-container">
+        <h1 class="intro-heading">The Best Experience Of <span class="intro-heading-span">Music</span> In 2022</h1>
+        <p>Music is life, music is love, music is everything.here you find list of all the upcoming concerts,global
```

Comment:
- [ ] Kindly add a top subtitle so as to match the given design

Your design
![image](https://user-images.githubusercontent.com/81932478/196936663-e4c90795-a444-4110-804c-0b358a780c50.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/196936754-701cb26a-4628-4c95-b9df-72cd45204f26.png)